dev-env/bin/daml-sdk-head

```diff
@@ -92,7 +92,7 @@ else
 fi
 
 cat > $DAML_HOME/bin/daml-head << EOF
-#!/bin/bash
+#!/bin/sh
```

Comment:
Didn’t seem worth spinning this out into its own PR. This is necessary for NixOS where `/bin/bash` is not a thing. We don’t require Bash here so no reason to not use this. Alternatively `/usr/bin/env bash` tends to work everywhere if you do need bash.